onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -464,6 +464,12 @@ def _topk_input_wrangler(
         matcher=lambda sample: len(sample.args[0]) == 0,
         reason="Empty perm is not supported",
     ),
+    skip(
+        "slice",
+        # kwargs {dim, start, end, step} is empty, we cannot give the default value
+        matcher=lambda sample: len(sample.kwargs) == 0,
```

Comment:
Could you help me understand why this is needed better?

Reply:
This was not introduced by this PR, it should be checked-in by another PR #304